localization/marker_tracking/include/marker_tracking/arxmlio.h

```diff
@@ -35,7 +36,7 @@ namespace marker_tracking {
   // 3 - TL
   typedef std::map<int, Eigen::Matrix<float, 4, 3>,
                    std::less<int>,
-                   Eigen::aligned_allocator<std::pair<int, Eigen::Vector4f> > >
+                   std::allocator<std::pair<int const, Eigen::Matrix<float, 4, 3>> > >
```

Comment:
what happened here? have to be careful https://eigen.tuxfamily.org/dox/group__TopicStlContainers.html, this bug might only show up on arm sometimes if I recall

Reply:
Thanks for the template, I followed it and didn't get compile errors (was following the std one before. I'm not sure how this bit worked before as it was trying to put a vector 4f into a matrix 4x3...